test/types/community/cursor.test-d.ts

```diff
@@ -65,12 +65,12 @@ typedCollection
   .map(x => x.name2 && x.age2);
 typedCollection.find({ name: '123' }, { projection: { age: 1 } }).map(x => x.tag);
 
-typedCollection.find().project({ name: 1 });
-typedCollection.find().project({ notExistingField: 1 });
-typedCollection.find().project({ max: { $max: [] } });
-
-// $ExpectType Cursor<{ name: string; }>
-typedCollection.find().project<{ name: string }>({ name: 1 });
+expectType<FindCursor<{ name: string }>>(typedCollection.find().project({ name: 1 }));
```

Comment:
is there a way to add at least one expectNotType test here?

Reply:
Also it looks like a number of assertions are failing in the pipeline

Reply:
Added two! pipeline should be clear, of course clarity is key so let me know if there's more I can add to cover more use cases.